customers.json

```diff
@@ -0,0 +1,19 @@
+{
+    "customers": [
```

Comment:
To can improve performance by making customers a dictionary where the customer id is the key. This way when searching for customer's info it's approximately O(1) to fetch the info (instead of O(n))